src/pages/Profile/Profile.jsx

```diff
@@ -1,5 +1,100 @@
-import React from "react";
-
+import axios from 'axios';
+import React, { useState } from 'react';
+import {
+  Footer,
+  Modal,
+  Navbar,
+  SideBarLeft,
+  SideBarRight,
+} from '../../components';
```

Comment:
It's better to use the convention of global imports first followed by local imports, makes code cleaner. So all react imports at the top, css at the last